runtime/src/chpl-privatization.c

```diff
@@ -18,17 +18,32 @@
  */
 
 #include "chplrt.h"
+#include "chpl-thread-local-storage.h"
 #include "chpl-privatization.h"
+#include "chpl-atomics.h"
 #include "chpl-mem.h"
 #include "chpl-tasks.h"
+#include "chpl-qsbr.h"
+#include <pthread.h>
+#include <stdbool.h>
```

Comment:
I think many of the above headers are not necessary now that we have chapel-qsbr.h